workshop.json

```diff
@@ -1,37 +1,47 @@
 {
-    "targets": {
-	"default": {
-	    "packages": [
+    "userenvs": [
+	{
+	    "name": "default",
+	    "requirements": [
 		"python36",
 		"python-redis"
```

Comment:
Just like we use a source package for default userenv instead of a binary package, I think we should do something similar for language-packages that can be installed via its native package-manager (in this case pip).  For example, this json as it sits would not work for ubi8, but if the requirements above had "python_redis" instead it would work (and I assume it would work with almost any userenv that had "python36" installed).

Reply:
I agree, but in this case I think that is what it is doing.  We probably need to come up with a more descriptive naming schema.  Something like "python-redis-pip" instead of "python-redis" or "python-redis-distro".  I believe that the idea of having the defaults be all build from source AND/OR install from language native tools is probably a decent idea, but at the moment I don't think that is the way all the JSON file are setup so we will have to deploy that scheme across them.